clients/src/main/java/org/apache/kafka/clients/ClientResponse.java

```diff
@@ -104,6 +135,50 @@ public long requestLatencyMs() {
         return latencyMs;
     }
 
+    public Logger getLogger() {
+        if (logContext == null) {
+            LogContext logContext = new LogContext("[" + requestHeader.toString() + "] ");
+            return logContext.logger(ClientResponse.class);
+        }
+        return logContext.logger(ClientResponse.class);
+    }
+
+    public void releaseBuffer() {
```

Comment:
should this be public? is there a use case for calling this NOT via decrementing the ref count ?